_posts/en/newsletters/2021-03-24-newsletter.md

```diff
@@ -0,0 +1,299 @@
+  the Bitcoin-Dev mailing list a description of a technique to
+  accomplish delegation using Bitcoin today.
+
+    Let's say Alice has `UTXO_A` and Bob has `UTXO_B`.  Alice creates a
+    partially signed transaction spending both `UTXO_A` and `UTXO_B`.
+    Alice signs for her UTXO using the sighash flag [SIGHASH_NONE][],
+    which prevents the signature from committing to any of the
+    transaction's outputs.  This gives the owner of the other input in
+    the transaction---Bob---unilateral control over the choice of
+    outputs, using his signature with the normal `SIGHASH_ALL` flag to
+    commit to those outputs and prevent anyone else from modifying the
+    transaction.  By using this dual-input `SIGHASH_NONE` trick, Alice
+    delegates to Bob the ability to sign for her UTXO.
+
+    This technique appears to be mainly of theoretical interest.  There
+    are other proposed delegation techniques---including graftroot,
+    [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify], and
+    [OP_CHECKSIGFROMSTACK][topic op_checksigfromstack]---that would
+    likely be superior in several ways, but only this technique is
+    currently usable on mainnet for anyone who wants to experiment with
+    it.
+
+- **Discussion of quantum computer attacks on taproot:**
+  the original Bitcoin software provided two ways to receive bitcoin:
+
+  - *Pay-to-Public-Key (P2PK)* implemented the simple and clear method
+    described in the [original Bitcoin paper][] of receiving bitcoins to
+    a public key and allowing those coins to be spent by a signature.  The
+    Bitcoin software used this by default when the public key material
+    could be handled entirely by software.
+
+  - *Pay-to-Public-Key-Hash (P2PKH)* added a layer of indirection,
+    receiving bitcoins to a hash digest that committed to the public key
+    to be used.  To spend the coins, the public key would need to be
+    published alongside the signature, making the 20 bytes dedicated to
+    the hash digest an overhead cost.  This was used by default when the
+    payment information might need to be handled by a person, e.g. an
+    address that could be copied and pasted.
+
+  Nakamoto never described why he implemented both methods, but it's
+  widely believed that he added the hash indirection in order to make
+  Bitcoin addresses smaller so that they could be communicated more
+  easily.  Public keys in the original Bitcoin implementation were 65
+  bytes, but address hashes were only 20 bytes.
+
+  In the decade since, there have been a number of developments.  To
+  make certain multisig protocols [simple and secure by default][bech32
+  address security], it was determined that scripts for multiparty
+  protocols should probably use a 32-byte commitment.  Bitcoin developers also
+  learned about previously known techniques that could compress a public
+  key down to 33 bytes---later [optimized][news48 oddness] to 32 bytes.
```

Comment:
It feels like saying that this was "later optimized to 32 bytes" is a bit premature, since it's just a proposal that isn't active on the Bitcoin network yet.

Perhaps this could be worded slightly differently to more clearly differentiate what is available for use now and what is part of a future proposal.